group-github-actions-minor-patch.json

```
@@ -0,0 +1,11 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "packageRules": [
+    {
+      "matchManagers": ["github-actions"],
+      "matchUpdateTypes": ["minor", "patch"],
+      "groupName": "github-actions non-major updates",
```

we probably don't have the choice but why not
```suggestion
      "groupName": "github-actions all-minor updates",
```
so we have consistency with the names:
- json file name = minor-patch.json
- grouSlug = all-minor-patch.

---

non major is everything but. so minor / patch

---

Im not too sure what your comment means :)

---

It's that between the name of the file "group-github-actions-minor-patch.json", the groupName: "github-actions non-major updates" and the groupSlug "github-actions-all-minor-patch", the naming is not consistent. It's minor but would prefere:
filename: "group-github-actions-minor-patch.json
groupName "github-action minor updates
groupSlug: "github-actions-all-minor-patch


only cosmetic...